x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/utils/utils.ts

```diff
@@ -31,3 +31,7 @@ export const ensureEngineExists =
 export const getByEntityTypeQuery = (entityType: EntityType) => {
   return `${entityEngineDescriptorTypeName}.attributes.type: ${entityType}`;
 };
+
+export const getEntitiesIndexName = (entityType: EntityType) => {
+  return `.entities.v1.latest.ea_${entityType}_entity_store`;
```

Comment:
This feels brittle 🤔 

The first thing I think we can improve is abstract our id section (`.ea_${entityType}_entity_store`) into a reusable function that's called here and on the hardcoded definition. 
But the bigger issue is if the Entity manager changes the index patterns. They have constants [here](https://github.com/elastic/kibana/blob/b103397b6d4d670c3bc676fa10dbb8ccb3d965bb/x-pack/plugins/observability_solution/entity_manager/common/constants_entities.ts), so maybe we can import those?

Reply:
I think we should create an alias in the future to make querying the entity store easier for users and to reduce users reliance on the exact naming of the backing indices.